analyzer/src/test/kotlin/managers/utils/NodeSupportTest.kt

```diff
@@ -24,6 +24,7 @@ import io.kotest.core.test.TestResult
 import io.kotest.matchers.collections.beEmpty
 import io.kotest.matchers.collections.shouldContainExactly
 import io.kotest.matchers.collections.shouldContainExactlyInAnyOrder
+import io.kotest.matchers.maps.beEmpty as beEmptyMap
```

Comment:
Nit: I was briefly wondering whether we should align the code exactly the other way around, but using `beEmpty()` has the advantage that no temporary empty collection are created. Maybe this is something you'd want to add to the commit message as part of the rationale.

Reply:
But `emptyList()` also creates no new instance but always returns the same object.

Reply:
Oh, right... so, no added benefit here then :sweat_smile: